addons/im_livechat/models/res_partner.py

```diff
@@ -80,3 +94,15 @@ def _compute_display_name(self):
             return
         for partner in self:
             partner.display_name = partner.name
+
+    def action_view_livechat_sessions(self):
+        action = self.env["ir.actions.act_window"]._for_xml_id("im_livechat.discuss_channel_action")
+        livechat_channel_ids = self.env['im_livechat.channel.member.history'].search([
+            ('partner_id', '=', self.id),
+            ('livechat_member_type', '=', 'visitor'),
+        ]).channel_id.ids
+        action["domain"] = expression.AND([
```

Comment:
One last remark, expression.AND is going to be deprecated, we should be using either Domain.AND or use Domain operators.